department-of-veterans-affairs/va.gov-cms
# VACMS-2043: Remove deprecated fields for Connect with us

## Description

Relates to #2043 . This is a clean rebuild of #7180 

## Testing done
Edited a CLP and saw the old field for connect with us was removed and the options for offices had been filtered to 4 main offices. Viewed the section taxonomy and saw the old connect with us fields were removed.

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/144931034-563238ce-72c9-4736-b4b9-a00c560c52a0.png)
![image](https://user-images.githubusercontent.com/8375335/144931179-fac9c8b9-093f-43cb-a34c-805a19528004.png)
![image](https://user-images.githubusercontent.com/8375335/147109731-766f7620-480b-4b7c-9ea2-4d0168a0dfd4.png)


## QA steps

As a cms editor I can
1. edit a campaign landing page (node/15051/edit) and see the old fields removed
2. see that the related_office field only has four options (VACO, VHA, VBA and NCA)
3. view the section taxonomy manage fields (/admin/structure/taxonomy/manage/administration/overview/fields) and see there are only two fields (description and product)
1. Then validate Acceptance Criteria from issue
- [x]  Delete social media link field, connect with us link field and email updates link text and url from section taxonomy.
- [x] Delete the 'add a connect with us panel' from CLP
- [x] Offices are filtered by a new content entity reference view (Main Offices) that shows only VACO, VHA, VBA and NCA 

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
